processor/src/main/kotlin/com/syouth/kmapper/processor/convertors/DataClassTypeConverter.kt

```diff
@@ -22,7 +23,10 @@ internal class DataClassTypeConverter(
 ) : TypeConvertor {
     override fun isSupported(from: KSType?, to: KSType, targetPath: PathHolder?): Boolean {
         if (from == null) return false
-        return from.isDataClass() && to.isDataClass() &&
+        return !( from.isSupportedCollectionType() ||
```

Comment:
Could you please place a TODO here saying "Think about better solution to classify classes responsible for convertion"